src/pages/Governance/hooks/useSubmitVote.ts

```diff
@@ -10,26 +10,16 @@ const useSubmitVote = () => {
   } = useSubmitTransaction();
 
   async function submitVote(
-    proposalId: number,
+    proposalId: string,
```

Comment:
should be string for now, once Petra move to use ABI interface we should be able to send a number
https://aptos-org.slack.com/archives/C03N83P7QUC/p1660945235254379